scipy/integrate/_quadrature.py

```diff
@@ -627,26 +651,111 @@ def simpson(y, x=None, dx=1.0, axis=-1, even='avg'):
         if x.shape[axis] != N:
             raise ValueError("If given, length of x along axis must be the "
                              "same as y.")
+
+    # even keyword parameter is deprecated
+    if 'even' in kwds:
```

Comment:
```suggestion
    if even is not None:
```

Reply:
`if 'even' in ['avg', 'first', 'last']` might be preferred.